README.md

```diff
@@ -28,6 +28,22 @@ To use ACA-Py you create a business logic controller that "talks to" an ACA-Py i
 This [checklist-style overview document](https://github.com/hyperledger/aries-cloudagent-python/blob/main/docs/features/SupportedRFCs.md) provides a full list of the features in ACA-Py.
 The following is a list of some of the core features needed for a production deployment, with a link to detailed information about the capability.
 
+## Releases
+
+Aca-py community provides periodic releases with new features and improvements. Additionally, certain releases are designated as long-term support (LTS) releases. Important fixes will be backported to the most recent LTS release, and to the prior LTS release during periods of LTS release overlap. For more details see the [LTS strategy](./docs/LTS-Strategy.md).
+
+Current LTS releases:
+- [0.12.x](https://github.com/hyperledger/aries-cloudagent-python/releases/tag/0.12.1) **Supported upto January 2025**
+- [0.11.x](https://github.com/hyperledger/aries-cloudagent-python/releases/tag/0.11.1) **Supported upto August 2024**
+
+Note that the third digit x corresponds to security patches to the LTS releases.
```

Comment:
Suggest we change the text above to be `[0.12-lts]` and `[0.11-lts]` and update this note to say that those are the= names of the branches from which the LTS patch releases are produced.